.github/workflows/conformance_with_build.yml

```diff
@@ -42,12 +42,12 @@ jobs:
             envoy-image: "envoyproxy/envoy:v1.22-latest"
             consul-k8s-version: "v0.49.0"
```

Comment:
is there a way of running this off of the master branch pre-release?

Reply:
Yeah I'd just have to add it as a trigger

Reply:
👍 I think doing `consul-k8s` at master would be good for ensuring that we're good in the agentless configuration pre-release.

Reply:
Actually I'm wrong, we can manually trigger it from main whenever we want already

![Screen Shot 2022-11-16 at 2 10 58 PM](https://user-images.githubusercontent.com/100602640/202284232-434a0751-7ddf-4568-adfe-af58d975e254.png)

Reply:
Still debugging why consul 1.14 isn't installing

Reply:
@sarahalsmiller Ah, I see, sorry, I meant to use the upcoming `consul-k8s` 1.0.0 release (i.e. their main branch) rather than 0.49.0 to install. The helm chart has changed _significantly_ for the agentless work.

Reply:
Ah I understand! That's still possible, I just need to update the ref value